projects/kamino-lending/index.js

```diff
@@ -0,0 +1,70 @@
+const { PublicKey, Keypair } = require('@solana/web3.js');
+const { getConnection, sumTokens } = require('../helper/solana');
+const { Program } = require('@project-serum/anchor');
+const kaminoIdl = require('./kamino-lending-idl.json');
+const { Token, TOKEN_PROGRAM_ID } = require('@solana/spl-token');
```

Comment:
I think you will have to remove this line as well

Reply:
https://github.com/DefiLlama/DefiLlama-Adapters/pull/8025/commits/e189b8f4f1d380f7cae2497970f713d56861c0bb

Reply:
This is how I did it ^

Reply:
I think it is an indirect dependency, but changed it to your code :)